README.md

```diff
@@ -1,13 +1,20 @@
-# Ring-Accept
+Ring-Accept-language
+====
 
-A Clojure library designed to ... well, that part is up to you.
+## Overview
+
+Ring-Accept-language provides a middleware that sets a hash that convert an accept-language in a request header.
 
 ## Usage
 
-FIXME
+Add a middleware after adding dependencies.
+
+    (defn some-middleware
+      [handler]
+      (wrap-accept-language handler))
 
 ## License
 
-Copyright © 2014 James Reeves
+Copyright © 2019 Studist Corporation
 
 Distributed under the MIT License, the same as Ring.
```

Comment:
Confirm with company if MIT License is ok

Reply:
Thank you for teaching me to write a license.
I've fixed it.